src/gaussjacobi.jl

```diff
@@ -367,39 +364,216 @@ function feval_asy1(n::Integer, α::Float64, β::Float64, t::AbstractVector, idx
     return vals, ders
 end
 
-function boundary(n::Integer, α::Float64, β::Float64, npts::Integer)
-# Algorithm for computing nodes and weights near the boundary.
+function asy2(n::Integer, α::Float64, β::Float64, npts::Integer)
```

Comment:
Could you please change the name to something more explicitly related to `gaussjacobi`, such as `_jacobi_asy2`?" (The same applies to `feval_asy2` etc.)

Reply:
The existing functions `asy1` and `feval_asy1` do not refer to the Jacobi aspect, and their names are taken directly from the Chebfun MATLAB source. For easy side-by-side reading with Chebfun, I think we should keep these names, as well as `asy2` and `feval_asy2` so that everything agrees.